mojo/core/node_controller.cc

```diff
@@ -508,6 +524,7 @@ void NodeController::SendBrokerClientInvitationOnIOThread(
   // called below.
 
   pending_invitations_.insert(std::make_pair(temporary_node_name, channel));
+  temp_node_name_ = temporary_node_name;
```

Comment:
We can access a node channel without this variable. See broker_castanets.h:126.

Reply:
Done.